easybuild/easyconfigs/m/MVAPICH2/MVAPICH2-2.1-GNU-5.1.0-2.25.eb

```diff
@@ -0,0 +1,23 @@
+name = 'MVAPICH2'
+version = '2.1'
+
+homepage = 'http://mvapich.cse.ohio-state.edu/overview/mvapich2/'
+description = "This is an MPI 3.0 implementation.  It is based on MPICH2 and MVICH."
+
+toolchain = {'name': 'GNU', 'version': '5.1.0-2.25'}
+
+source_urls = ['http://mvapich.cse.ohio-state.edu/download/mvapich/mv2/']
+sources = [SOURCELOWER_TAR_GZ]
+
+preconfigopts = 'unset F90 && unset FC && unset F90FLAGS && unset FCFLAGS && '
```

Comment:
this is futile, the MVAPICH2 easyblock checks for `$FC` and `$F90`, prior to even looking at `(pre)configopts`

something is setting `$FC`, while `eb` itself defines `$F90` via the toolchain support